structured_external_storage_all/models/ir_attachment_sync_rule.py

```diff
@@ -0,0 +1,80 @@
+# -*- coding: utf-8 -*-
+# © 2018 Sunflower IT (http://sunflowerweb.nl)
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+from odoo import api, fields, models, _
+from odoo.exceptions import UserError, ValidationError
+from odoo.tools.safe_eval import safe_eval
+
+
+class AttachmentSyncRule(models.Model):
+    _name = 'ir.attachment.sync.rule'
+    _description = 'Define Specific Rules for Cloud Sync'
+
+    @api.model
+    def _get_model_selection(self):
+        """Get models and their names."""
+        model_objs = self.env["res.request.link"].search([
+            ("ext_attachment_sync", "=", True)], order="name")
+        return [(m.object, m.name) for m in model_objs]
+
+    name = fields.Char('Name', required=True)
+    source_model = fields.Many2one(
+        'res.request.link', 'Source Model', required=True,
+        domain=[('ext_attachment_sync', '=', True)])
+    sync_type = fields.Selection(
+        string='Sync Type', related='location_id.protocol', readonly=True)
+    frequency = fields.Integer('Frequency in Days', required=True)
+    sync_old_attachments = fields.Boolean(
+        'Sync Pre-existing Records',
+        help="Only useful if you want to Upload existing attachments before "
+        "the installation of this module")
+    last_sync_date = fields.Datetime('Last Sync Date')
+    location_id = fields.Many2one(
+        'external.file.location', 'Sync Name', required=True)
+    domain = fields.Char(
+        'Domain', default=[(1,'=',1)],
+        help='Conditions that will apply to the '
+        'select model. E.g [(amount, '>', 500, )]')
```

Comment:
Here you should use `"` surrounding the string, otherwise the `'` you use to surround the `>` will be seen as the end of the string